input/resources/au-relatedperson.xml

```diff
@@ -68,10 +89,15 @@
     <element id="RelatedPerson.extension:recordedSexOrGender">
       <path value="RelatedPerson.extension"/>
       <sliceName value="recordedSexOrGender"/>
+      <short value="Recorded sex or gender type shall be a member of the Common AU Recorded Sex or Gender Type value set if any codes within that value set can apply.Recorded sex or gender source document type shall be a member of the Common AU Recorded Sex or Gender (RSG) Source Document Type value set if any codes within that value set can apply. Recorded sex or gender jurisdiction shall be a member of the Common AU Recorded Sex or Gender (RSG) Source Document Jurisdiction value set if any codes within that value set can apply"/>
```

Comment:
I will accept for the purposes of being consistent with the entries to get this content in. But having a short that is a repetition of invariant feedback is not good. Spec feedback should be raised.